language-sql.md

```diff
@@ -2,12 +2,57 @@
 
 ### Minimum Requirements
 
-- MySQL keywords in uppercase
-- Column names in lowercase
+- MySQL keywords in uppercase.
+
+    Example: 
+    ```SQL
+    VALID
+        UPDATE `consumer` SET `password` = MD5(`password`);
+
+    NOT VALID:
+        update `consumer` set `password` = md5(`password`);
+    ```
```

Comment:
? Proposal to keep the code only and emphasize their validity:

- MySQL keywords in uppercase.

    **Valid**
    ```SQL
    UPDATE `consumer` SET `password` = MD5(`password`);
    ```
    **Not valid**
    ```sql
    update `consumer` set `password` = md5(`password`);
    ```